pkg/handlers/config.go

```diff
@@ -301,6 +318,26 @@ func (h *Handler) CurrentAppConfig(w http.ResponseWriter, r *http.Request) {
 	JSON(w, http.StatusOK, CurrentAppConfigResponse{Success: true, ConfigGroups: renderedConfig.Spec.Groups})
 }
 
+func isVersionConfigEditable(app *apptypes.App, sequence int64) (bool, error) {
+	// Only latest and currently deployed versions can be edited
+	if app.CurrentSequence == sequence {
+		return true, nil
+	}
+
+	downstreams, err := store.GetStore().ListDownstreamsForApp(app.ID)
+	if err != nil {
+		return false, errors.Wrap(err, "failed to get downstreams")
+	}
+
+	for _, d := range downstreams {
+		if d.CurrentSequence == sequence {
```

Comment:
```suggestion
		if d.ParentSequence == sequence {
```

Reply:
There's no ParentSequence in this one.  I think that is the right field.  It's just sequence: https://github.com/replicatedhq/kots/blob/051058090f62305445d9761930eb9b0775839774/pkg/version/version.go#L104